website/docs/r/blockstorage_volume_v2.html.markdown

```diff
@@ -69,7 +69,7 @@ The following arguments are supported:
     Changing this creates a new volume.
 
 * `volume_type` - (Optional) The type of volume to create.
-    Changing this creates a new volume.
+    Changing this creates a new volume. Currently, the value can be SSD, SAS, SATA, co-p1, or uh-l1. Changing this creates a new volume.
```

Comment:
"Changing this  creates a new volume." was duplicated.

Reply:
usually, the documentation specifies under every attribute, which changes lead to a new creation. I can not see a duplication here. I would change the current working to the new volume - Update documentatio:
(Optional) Currently, the value can be SSD (ultra-I/O disk type), SAS (high I/O disk type), or SATA (common I/O disk type) OTC-API

Reply:
The duplication should be a non-issue now, updated the documentation to a more extensive and readable way. 
/Resolved